apps/concierge_site/assets/mjml/notification.mjml

```diff
@@ -99,10 +99,10 @@
     <mj-raw>
       <!-- Footer -->
     </mj-raw>
-    <mj-section padding="0">
+    <mj-section padding="0" css-class="footer">
       <mj-column>
         <mj-text font-size="12px" font-family="'Helvetica Neue', Helvetica, Arial, sans-serif" padding-left="0" padding-right="0">
-          <p>
+          <p role="contentinfo">
```

Comment:
In an earlier commit I was using an [mj-html-attributes](https://documentation.mjml.io/#mj-html-attributes) tag to put the `role` attribute on the enclosing `<div>`, but the build complained. MJML added that tag in 4.7.0, so maybe our library is a bit out of date?